listener.go

```diff
@@ -127,15 +136,25 @@ func (h *apigeeSyncHandler) processChangeList(changes *common.ChangeList) {
 		}
 	*/
 
+	// insert
 	for i := range insertedDeployments {
 		go h.bundleMan.queueDownloadRequest(&insertedDeployments[i])
 	}
 
-	// clean up old bundles
+	// update
+	for i := range updatedNewBlobs {
+		go h.bundleMan.enqueueRequest(h.bundleMan.makeDownloadRequest(updatedNewBlobs[i]))
```

Comment:
is it better if we split this routine to two steps
1. update db 
2. download 

step 1 will be quick and step 2 can be delayed and there is different exception handling for both. Basically retires are required only for step 2